exercises/practice/etl/.meta/reference/etl.bal

```diff
@@ -0,0 +1,20 @@
+public function transform(map<string[]> old) returns map<int> {
+    map<int> transformed = {};
```

Comment:
@glennj AFAICT you can only have a map with `string` keys. It doesn't really matter much for this exercise, so I just went with it.